### ISSUE TITLE
FreeBSD support is broken

### ISSUE THREAD

**Issue by imp1sh** [willshersystems/ansible-sshd#226]
It seems the variables are incomplete.

```
fatal: [ansiblerx1.example.com]: FAILED! => {"msg": "The task includes an option with an undefined variable. The error was: {{ __sshd_packages }}: '__sshd_packages' is undefined\n\nThe error appears to be in '/usr/home/jochen/.ansible/roles/willshersystems.sshd/tasks/install.yml': line 7, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- name: Install ssh packages\n  ^ here\n"}
```

**Comment by richm** [comment]
> It seems the variables are incomplete.
> 
> ```
> fatal: [ansiblerx1.example.com]: FAILED! => {"msg": "The task includes an option with an undefined variable. The error was: {{ __sshd_packages }}: '__sshd_packages' is undefined\n\nThe error appears to be in '/usr/home/jochen/.ansible/roles/willshersystems.sshd/tasks/install.yml': line 7, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- name: Install ssh packages\n  ^ here\n"}
> ```

Did it used to work?  If so, do you know what was the last version of ansible-sshd that worked with FreeBSD?

**Comment by imp1sh** [comment]
I don't know if it ever worked. I'm on v0.18.2

**Comment by richm** [comment]
> I don't know if it ever worked. I'm on v0.18.2 The role suggests it supports FreeBSD though.

Yeah, not sure when it was broken, probably when we introduced `__sshd_packages`.  Looks like `vars/FreeBSD.yml` and `vars/OpenBSD.yml` do not define this.  I'm not familiar with bsd and I don't know what packages should be installed.

**Comment by imp1sh** [comment]
I will give it a try and if I figure things out I will do a PR.

**Comment by imp1sh** [comment]
As FreeBSD doesn't need packages you can set the variable just to an empty list, i.e. []
I guess you won't need a pull request for that one, right?

**Comment by richm** [comment]
> As FreeBSD doesn't need packages you can set the variable just to an empty list, i.e. [] I guess you won't need a pull request for that one, right?

Does this work?  https://github.com/willshersystems/ansible-sshd/pull/227

**Comment by imp1sh** [comment]
For FreeBSD it does, for OpenBSD I cannot tell as I don't run it.

**Comment by Jakuje** [comment]
This was fixed with  #227. Thank you for the report!

**Comment by infinite-ops** [comment]
> For FreeBSD it does, for OpenBSD I cannot tell as I don't run it.

Confirmed this works for OpenBSD also - thankyou
